panel/src/components/Sections/ModelsSection.vue

```diff
@@ -91,6 +95,39 @@ export default {
 		buttons() {
 			let buttons = [];
 
+			if (this.isSelecting) {
+				buttons.push({
+					disabled: this.selected.length === 0,
+					icon: "trash",
+					text: this.$t("delete") + ` (${this.selected.length})`,
+					theme: "negative",
+					click: () => {
+						this.$panel.dialog.open({
+							component: "k-remove-dialog",
+							props: {
+								text: this.confirmDeleteSelectedMessage
```

Comment:
```suggestion
								text: this.$t(`${this.type}.delete.confirm.selected`, {
										count: this.selected.length
								});
```

The computed prop is only ever used here. I'd prefer to keep it here and not add an additional computed prop.